### PR TITLE
Set CommissioningModeInitialStepsHint to 1 by default for standard flow

### PR DESCRIPTION
**Pull request by akarabashov**

### CODE REVIEW

File: docs/transactions.md [hunk 1]
```diff
@@ -260,7 +260,8 @@ All non-edited fields remain the same.
   - productURL: `optional(string)` - URL that contains product specific web page that contains details for the device model.  
   - lsfURL: `optional(string)` - URL to the Localized String File of this product.
   - schemaVersion: `optional(uint16)` - Schema version to support backward/forward compatability(default 0)
-- lsfRevision: `optional(uint32)` - LsfRevision is a monotonically increasing positive integer indicating the latest available version of Localized String File.
+  - lsfRevision: `optional(uint32)` - LsfRevision is a monotonically increasing positive integer indicating the latest available version of Localized String File.
+  - commissioningModeInitialStepsHint: `optional(uint32)` - commissioningModeInitialStepsHint SHALL identify a hint for the steps that can be used to put into commissioning mode a device that has not yet been commissioned. This field is a bitmap with values defined in the Pairing Hint Table. For example, a value of 1 (bit 0 is set) indicates that a device that has not yet been commissioned will enter Commissioning Mode upon a power cycle.
```
Comment (ashcherbakov) [inline docs/transactions.md:264]: Should we mention that it can not be updated to zero?

File: x/model/handler_test.go [hunk 2]
```diff
@@ -170,7 +170,47 @@ func TestHandler_AddModel(t *testing.T) {
...
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			setup := Setup(t)
+
+			// add new model
+			msgCreateModel := NewMsgCreateModel(setup.Vendor)
+			msgCreateModel.CommissioningModeInitialStepsHint = tc.expectedCommissioningModeInitialStepsHint
```
Comment (ashcherbakov) [inline x/model/handler_test.go:199]:
```suggestion
			msgCreateModel.CommissioningModeInitialStepsHint = tc.commissioningModeInitialStepsHint
```

File: docs/transactions.md (hunk 1 above)
Comment (akarabashov) [reply docs/transactions.md]: Done

File: x/model/handler_test.go (hunk 2 above)
Comment (akarabashov) [reply x/model/handler_test.go]: Done

**Review by ashcherbakov — APPROVED** [review]

**Pull request merged**